controllers/infra/drivers/aws/price.go

```diff
@@ -8,5 +8,5 @@ type Price struct {
 // QueryPrice query infra price/hour
 func (pReq Price) QueryPrice(infra *v1.Infra) (int64, error) {
 	//TODO implement me
-	panic("implement me")
+	return 1, nil
```

Comment:
func (infra *v1.Infra) QueryPrice() (int64, error) is better?

Reply:
ok

Reply:
this function completed